server/service/integration_core_test.go

```diff
@@ -341,7 +341,7 @@ func (s *integrationTestSuite) TestVulnerableSoftware() {
 	assert.Len(t, lsResp.Software[0].Vulnerabilities, 1)
 	assert.Equal(t, 1, lsResp.Software[0].HostCount)
 
-	s.DoJSON("GET", "/api/v1/fleet/software", lsReq, http.StatusOK, &lsResp, "vulnerable", "true", "order_key", "host_count", "order_direction", "desc")
+	s.DoJSON("GET", "/api/v1/fleet/software", lsReq, http.StatusOK, &lsResp, "vulnerable", "true", "order_key", "host_count,id", "order_direction", "desc")
```

Comment:
I feel we need a test on the `mysql` datastore side that checks for the expected ordering of multiple fields.